src/api/axiosClient.ts

```diff
@@ -5,7 +5,7 @@ const getDefaultClient = () => {
   const client = axios.create();
   client.interceptors.request.use((cfg) => ({
     ...cfg,
-    url: `${process.env.REACT_APP_API_ROOT}${cfg.url}`,
+    url: `${window.location.origin}${process.env.REACT_APP_API_ROOT}${cfg.url}`,
```

Comment:
This will not work since the `REACT_APP_API_URL` is being provided as a full URL, means in a format like: `http://10.19.130.2:6000`